aim/api/resource.py

```diff
@@ -291,6 +291,225 @@ def __init__(self, **kwargs):
+
+    _aci_mo_name = 'spanVSrc'
+    _tree_parent = SpanVsourceGroup
+
+    def __init__(self, **kwargs):
+        super(SpanVsource, self).__init__({'dir': 'both',
+                                           'monitored': False},
+                                          **kwargs)
+
+
+class SpanSrcVport(AciResourceBase):
+    """Resource representing a Relation to SPAN Source to Port.
+
+    Identity attributes are name of Source Group name, source name
+    and src_path of the VSPAN session.
+    """
+
+    identity_attributes = t.identity(
+        ('vsg_name', t.name),
+        ('vs_name', t.name),
+        ('src_path', t.string()))
+    other_attributes = t.other(
+        ('monitored', t.bool))
+
+    _aci_mo_name = 'spanRsSrcToVPort'
+    _tree_parent = SpanVsource
+
+    def __init__(self, **kwargs):
+        super(SpanSrcVport, self).__init__({'monitored': False},
+                                           **kwargs)
+
+
+class SpanVdestGroup(AciResourceBase):
+    """Resource representing a ERSPAN VDest Group in ACI.
+
+    Identity attributes are RNs for ERSPAN VDest Group.
+    """
+
+    identity_attributes = t.identity(
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name))
+
+    _aci_mo_name = 'spanVDestGrp'
+    _tree_parent = Infra
+
+    def __init__(self, **kwargs):
+        super(SpanVdestGroup, self).__init__({'monitored': False},
+                                             **kwargs)
+
+
+class SpanVdest(AciResourceBase):
+    """Resource representing a ERSPAN VDest in ACI.
+
+    Identity attributes are RNs for ERSPAN VDest.
+    """
+
+    identity_attributes = t.identity(
+        ('vdg_name', t.name),
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name))
+
+    _aci_mo_name = 'spanVDest'
+    _tree_parent = SpanVdestGroup
+
+    def __init__(self, **kwargs):
+        super(SpanVdest, self).__init__({'monitored': False},
+                                        **kwargs)
+
+
+class SpanVepgSummary(AciResourceBase):
+    """Resource representing a ERSPAN Destination VEpg Summary in ACI.
+
+    Identity attributes are RNs for ERSPAN Destination VEpg Summary.
+    """
+
+    identity_attributes = t.identity(
+        ('vdg_name', t.name),
+        ('vd_name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name),
+        ('dst_ip', t.string()),
+        ('flow_id', t.string()),
+        ('ttl', t.string()),
+        ('mtu', t.string()))
+
+    _aci_mo_name = 'spanVEpgSummary'
+    _tree_parent = SpanVdest
+
+    def __init__(self, **kwargs):
+        super(SpanVepgSummary, self).__init__({'monitored': False,
+                                               'dst_ip': '',
+                                               'flow_id': '1',
+                                               'ttl': '64',
```

Comment:
Not sure why we put the default value for ttl=64 and mtu=1518 here.....

Reply:
Thats because if we check on the UI and configure from the UI, those are the default values set by apic. So I used them.

Reply:
ok